lerna-debug.log

```diff
@@ -0,0 +1,11 @@
+0 silly argv {
```

Comment:
Looks like this file should not be committed.

Reply:
Huh, silly me 😝